lib/node_modules/@stdlib/stats/base/dists/studentized-range/cdf/benchmark/benchmark.js

```diff
@@ -21,27 +21,35 @@
 // MODULES //
 
 var bench = require( '@stdlib/bench' );
-var randu = require( '@stdlib/random/base/randu' );
 var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var uniform = require( '@stdlib/random/base/uniform' );
+var Float64Array = require( '@stdlib/array/float64' );
 var pkg = require( './../package.json' ).name;
 var cdf = require( './../lib' );
 
```

Comment:
```suggestion
var cdf = require( './../lib' );


```

Header comments should be preceded by two empty lines.